packages/framework/fluid-static/src/rootDataObject.ts

```diff
@@ -135,6 +139,10 @@ export class RootDataObject
 		const obj = this.runtime.createChannel(undefined, factory.type);
 		return obj as unknown as T;
 	}
+
+	public submitSignal<T>(signalName: string, payload?: Jsonable<T>) {
```

Comment:
I see you have payload optional here but not on the FluidContainer?  Probably want to have those match one way or the other (optional makes sense to me).

Also I like the Jsonable usage here, maybe we should consider putting that typing on the DataStoreRuntime too.